src/notification.rs

```diff
@@ -133,7 +148,7 @@ impl fmt::Debug for Notification {
                self.get_event(),
                self.get_button_id(),
                self.get_scene_id(),
-               self.get_notification(),
+               self.get_notification_code(),
```

Comment:
So I played with this some more and now I get starting/waiting when I do the add/remove node and then get inprogress/completed when done.

So we could remember the last state message received from the controller and return that.

I'll do a separate PR for that.